pkgs/by-name/sb/sbd/package.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sbd";
+  version = "1.36";
```

```suggestion
  version = "1.37";
```
is the latest version

---

Did you read my post above? On packetstorm the latest is 1.36. The 1.37 is available only on Kali repo and on the non-available website of its devs.

---

Is it available on archive.org?

---

> Is it available on archive.org?

Not sure if using archive.org could be a clean way. The best approach could be downloading the 1.37 and upload it on packetstormsecurity if possible